### DISCUSSION TITLE
why i want install this package i get this Error !! i thinck some things is wrong in this packeage

### DISCUSSION THREAD

**Discussion by isrezaei** [ashutosh1919/react-awesome-loaders#16] [General]
npm ERR! code 1
npm ERR! path C:\Users\isrez\Desktop\New folder\node_modules\react-awesome-loaders\node_modules\node-sass
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c node scripts/build.js
npm ERR! Building: C:\Program Files\nodejs\node.exe C:\Users\isrez\Desktop\New folder\node_modules\react-awesome-loaders\node_modules\node-gyp\bin\node-gyp.js rebuild --verbose --libsass_ext= --libsass_cflags= --libsass_ldflags= --l
ibsass_library=
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp verb cli [
npm ERR! gyp verb cli   'C:\\Program Files\\nodejs\\node.exe',
npm ERR! gyp verb cli   'C:\\Users\\isrez\\Desktop\\New folder\\node_modules\\react-awesome-loaders\\node_modules\\node-gyp\\bin\\node-gyp.js',
npm ERR! gyp verb cli   'rebuild',
npm ERR! gyp verb cli   '--verbose',
npm ERR! gyp verb cli   '--libsass_ext=',
npm ERR! gyp verb cli   '--libsass_cflags=',
npm ERR! gyp verb cli   '--libsass_ldflags=',
npm ERR! gyp verb cli   '--libsass_library='
npm ERR! gyp verb cli ]
npm ERR! gyp info using node-gyp@3.8.0
npm ERR! gyp info using node@16.15.1 | win32 | x64
npm ERR! gyp verb command rebuild []
npm ERR! gyp verb command clean []
npm ERR! gyp verb clean removing "build" directory
npm ERR! gyp verb command configure []
npm ERR! gyp verb check python checking for Python executable "python2" in the PATH
npm ERR! gyp verb `which` failed Error: not found: python2
npm ERR! gyp verb `which` failed     at getNotFoundError (C:\Users\isrez\Desktop\New folder\node_modules\react-awesome-loaders\node_modules\which\which.js:13:12)
npm ERR! gyp verb `which` failed     at F (C:\Users\isrez\Desktop\New folder\node_modules\react-awesome-loaders\node_modules\which\which.js:68:19)
npm ERR! gyp verb `which` failed     at E (C:\Users\isrez\Desktop\New folder\node_modules\react-awesome-loaders\node_modules\which\which.js:80:29)
npm ERR! gyp verb `which` failed     at C:\Users\isrez\Desktop\New folder\node_modules\react-awesome-loaders\node_modules\which\which.js:89:16
npm ERR! gyp verb `which` failed     at C:\Users\isrez\Desktop\New folder\node_modules\isexe\index.js:42:5
npm ERR! gyp verb `which` failed     at C:\Users\isrez\Desktop\New folder\node_modules\isexe\windows.js:36:5
npm ERR! gyp verb `which` failed     at FSReqCallback.oncomplete (node:fs:198:21)
npm ERR! gyp verb `which` failed  python2 Error: not found: python2
npm ERR! gyp verb `which` failed     at getNotFoundError (C:\Users\isrez\Desktop\New folder\node_modules\react-awesome-loaders\node_modules\which\which.js:13:12)
npm ERR! gyp verb `which` failed     at F (C:\Users\isrez\Desktop\New folder\node_modules\react-awesome-loaders\node_modules\which\which.js:68:19)
npm ERR! gyp verb `which` failed     at E (C:\Users\isrez\Desktop\New folder\node_modules\react-awesome-loaders\node_modules\which\which.js:80:29)
npm ERR! gyp verb `which` failed     at C:\Users\isrez\Desktop\New folder\node_modules\react-awesome-loaders\node_modules\which\which.js:89:16
npm ERR! gyp verb `which` failed     at C:\Users\isrez\Desktop\New folder\node_modules\isexe\index.js:42:5
npm ERR! gyp verb `which` failed     at C:\Users\isrez\Desktop\New folder\node_modules\isexe\windows.js:36:5
npm ERR! gyp ERR! stack Error: Can't find Python executable "python", you can set the PYTHON env variable.
npm ERR! gyp ERR! stack     at PythonFinder.failNoPython (C:\Users\isrez\Desktop\New folder\node_modules\react-awesome-loaders\node_modules\node-gyp\lib\configure.js:484:19)
npm ERR! gyp ERR! stack     at PythonFinder.<anonymous> (C:\Users\isrez\Desktop\New folder\node_modules\react-awesome-loaders\node_modules\node-gyp\lib\configure.js:509:16)
npm ERR! gyp ERR! stack     at callback (C:\Users\isrez\Desktop\New folder\node_modules\graceful-fs\polyfills.js:306:20)
npm ERR! gyp ERR! stack     at FSReqCallback.oncomplete (node:fs:198:21)
npm ERR! gyp ERR! System Windows_NT 10.0.19044
npm ERR! gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\isrez\\Desktop\\New folder\\node_modules\\react-awesome-loaders\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild" "--verbose" "--libsass_ext=" "--libsass
_cflags=" "--libsass_ldflags=" "--libsass_library="
npm ERR! gyp ERR! cwd C:\Users\isrez\Desktop\New folder\node_modules\react-awesome-loaders\node_modules\node-sass
npm ERR! gyp ERR! node -v v16.15.1
npm ERR! gyp ERR! node-gyp -v v3.8.0
npm ERR! gyp ERR! not ok
npm ERR! Build failed with error code: 1

**Comment by ashutosh1919** [comment]
@isrezaei I think the node version that you are using is not compatible. Also, I think you have not installed python. Node uses Python2 to work correctly.
